repose-aggregator/components/filters/valkyrie-authorization/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -61,18 +64,11 @@ class ValkyrieAuthorizationFilter @Inject()(configurationService: ConfigurationS
     configurationService.unsubscribeFrom(configurationFile, this)
   }
 
-  trait ValkyrieResult
-
-  case class UserInfo(tenantId: String, contactId: String) extends ValkyrieResult
-  case class UserPermissions(roles: Vector[String], devices: Vector[DeviceToPermission]) extends ValkyrieResult
-  case class DeviceToPermission(device: Int, permission: String)
-  case class ResponseResult(statusCode: Int, message: String = "") extends ValkyrieResult
-
   override def doFilter(servletRequest: ServletRequest, servletResponse: ServletResponse, filterChain: FilterChain): Unit = {
     val mutableHttpRequest = MutableHttpServletRequest.wrap(servletRequest.asInstanceOf[HttpServletRequest])
     val mutableHttpResponse = MutableHttpServletResponse.wrap(mutableHttpRequest, new HttpServletResponseWrapper(servletResponse.asInstanceOf[HttpServletResponse]))
 
-    def nullOrWhitespace(str: Option[String]): Option[String] = str.map { _.trim }.filter { !"".equals(_) }
+    def nullOrWhitespace(str: Option[String]): Option[String] = str.map(_.trim).filter(!"".equals(_))
```

Comment:
Albeit the parens are the way I normally see this, is there any functional difference in these two notations?